### PR TITLE
VIH-7436 edit hearing start time

### PR DESCRIPTION
**Pull request by surfsquid**
https://tools.hmcts.net/jira/browse/VIH-7436

Changes to other_information property in hearing to send and receive data as a pipe delimited string

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by npoxon — CHANGES_REQUESTED** [review]
Please don't merge this without AC tests

**Review by levibutler — APPROVED** [review]
Nice :)
Nick would like to wait for AC tests tho please

**Review by npoxon — DISMISSED** [review]
LGTM

File: AdminWebsite/AdminWebsite/ClientApp/src/app/services/pipe-stringifier.service.spec.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    it('should be created', () => {
+        expect(service).toBeTruthy();
+    });
+
+    it('should encode an object to a string', () => {
+        expect(service.encode(testObject)).toEqual(testString);
+    });
+
+    it('should decode a string into an object', () => {
+        expect(service.decode(testString)).toEqual(testObject);
```
Comment (levibutler) [inline AdminWebsite/AdminWebsite/ClientApp/src/app/services/pipe-stringifier.service.spec.ts:23]: could we test a couple more not so happy paths?
`|name||height|100` // With empty
`|name|null|height|100` // With null
`|prop1|1|prop2|2|prop3|3` // 3 properties in values should parse to string
`|name|test|height|` // missing height
`|otherinfo|containingapipe|likethis`
null or empty

**Review by levibutler — APPROVED** [review]

**Pull request merged**